test/indexSpec.js

```diff
@@ -24,6 +24,10 @@ describe('constructor', () => {
     assert.isObject(new PluginVirtual(options))
   })
 
+  it('should export the Token methods from the module', () => {
```

Comment:
Nevertheless, there should be at least one test that says "it should return ..." and compares the value returned against a fixture